src/shieldtest.js

```diff
@@ -0,0 +1,233 @@
+"use strict";
+
+import { map } from "./americana.js";
+import * as shield from "./js/shield.js";
+import { shields } from "./js/shield_defs.js";
+import * as gfx from "./js/screen_gfx.js";
+
+const once = (emitter, name, { signal } = {}) =>
+  new Promise((resolve, reject) => {
+    emitter.once(name, resolve);
+    signal?.addEventListener("abort", reject);
+  });
+
+await once(map, "load");
+
+let networks = [
```

Comment:
Would it make sense to load these programmatically from the array that we generate in shield_defs.js?

Reply:
That was my initial plan and is what this line does:
https://github.com/ZeLonewolf/openstreetmap-americana/blob/903323c53e44796421fdf602d20d18dde05192d0/src/shieldtest.js#L188-L189

I replaced it with a custom list because it ends up including a lot of duplicate graphic since many networks values use the same shield:
![Screen Shot 2022-08-22 at 11 15 05 PM](https://user-images.githubusercontent.com/281482/186244482-46f2dae8-5351-44fd-bbae-9bff58e05b2b.png)

I left the commented out line there so we could toggle it back on when you want to see _all_ the networks, not just all the shields.

Reply:
The full list also can take quite a long time to load and it crashed my browser tab at times.

Reply:
Ah yes, counties are dumb.